quinn-proto/src/tests/util.rs

```diff
@@ -375,21 +375,17 @@ impl Write for TestWriter {
     }
 }
 
-pub fn server_config() -> ServerConfig {
-    let key = CERTIFICATE.serialize_private_key_der();
-    let cert = CERTIFICATE.serialize_pem().unwrap();
-
-    let mut crypto = crypto::ServerConfig::new();
-    Arc::make_mut(&mut crypto)
-        .set_single_cert(
-            rustls::internal::pemfile::certs(&mut cert.as_bytes()).unwrap(),
-            rustls::PrivateKey(key.to_vec()),
-        )
-        .unwrap();
-    ServerConfig {
-        crypto,
-        ..Default::default()
-    }
+pub fn server_config(cert: Certificate, key: PrivateKey) -> ServerConfig {
+    ServerConfig::with_single_cert(CertificateChain::from_certs(vec![cert]), key).unwrap()
+}
+
+pub fn default_server_config() -> ServerConfig {
```

Comment:
nit: If we named this `server_config`, and the above `server_config_with_cert` or something, it'd reduce churn

Reply:
Done.